boards/common/blxxxpill/Makefile.include

```diff
@@ -8,15 +8,15 @@ ifeq (dfu-util,$(PROGRAMMER))
 
   # Leave some space at the beginning of the flash for the bootloader
   ROM_OFFSET ?= 0x2000
-
+  FFLAGS_OPTS = --dfuse-address $(FLASH_ADDR):leave
   # If using STM32duino bootloader, this change is necessary.
   # Bootloader available at
   # github.com/rogerclarkmelbourne/STM32duino-bootloader/tree/master/binaries
   ifeq (stm32duino,$(BOOTLOADER))
-    FFLAGS_OPTS ?= -R
```

Comment:
Here I wouldn't configure `FFLAGS` here but rather extend`FFLAGS_OPTS` as with other board configuration. What about something like:

```
FFLAGS_OPTS += --reset
```

And you remove FFLAGS use from this file. It will cleaner and simpler.

Reply:
No sure to understand what you want but I pushed a fix.

Reply:
You don't need to set --device, --download and --alt options here. They are already set in the common `dfu-util.inc.mk` file.
To remain the same as in master, you have to set `--reset` only with the stm32duino bootloader.

Reply:
So that would just be:

```mk
FFLAGS_OPTS ?= --dfuse-address $(FLASH_ADDR):leave
```
under ROM_OFFSET, and

```mk
FFLAGS_OPTS += --reset
```
in the `ifeq (stm32duino,$(BOOTLOADER))` block. The comment `Flashing may be easier...` could also be moved before the `FFLAGS_OPTS += --reset` line as it seems to be related to it.

Reply:
Pushed a fix. I think this is what you want :)